assets/js/views/_paymentCategories.js

```diff
@@ -0,0 +1,63 @@
+OpenDisclosure.PaymentCategoryView = Backbone.View.extend({
+  initialize: function(options) {
+    this.options = options;
+    _.bindAll(this, 'displayDetails');
+    this.render();
+  },
+
+  render: function() {
+    var attributes = this.options.attributes;
+    this.$el.empty();
+    this.$el.append('<h2>Total Payments by Category</h2>');
+    // Create the data table.
+    var data = new google.visualization.DataTable();
+    this.data = data;
+    data.addColumn('string', 'Category');
+    data.addColumn('number', 'Amount');
+    data.addColumn('string', 'Code');
+    _.each(this.collection, function( c ){
+      data.addRow([c.attributes.text, c.attributes.amount, c.attributes.code]);
```

Comment:
Backbone purists would say you should use `c.get('text')` instead of `c.attributes.text`, but whatever.